### PR TITLE
Collapsible Side Bar

### PR DESCRIPTION
**Pull request by jahn96**
## Tl;dr
* Collapsible side bar with a button

## GIF
Kapture 2021-08-05 at 10 34 30

### CODE REVIEW

File: src/components/VisualizationScreen.tsx [hunk 1]
```diff
@@ -20,6 +22,16 @@ const bifrostWidgetCss = css`
   @media screen and (max-width: 1300px) {
     display: block;
   }
+
+  .side-bar-collapsible-button {
+    transition: transform 0.4s ease-in-out;
+    transform-origin: 35% 35%;
```
Comment (Waidhoferj) [inline src/components/VisualizationScreen.tsx:28]: I think the rotation is still a bit off center
```suggestion
    transform-origin: 40% 35%;```

**Review by Waidhoferj — CHANGES_REQUESTED** [review]
Just one comment

File: src/components/VisualizationScreen.tsx (hunk 1 above)
Comment (jahn96) [reply src/components/VisualizationScreen.tsx:28]: Thanks!

**Review by Waidhoferj — APPROVED** [review]
LGTM

**Pull request merged**